packages/test/src/test-default-workflow.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Test usage of a default workflow handler
+ */
+import test from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import { WorkflowClient } from '@temporalio/client';
+import { RUN_INTEGRATION_TESTS, Worker } from './helpers';
+import { existing } from './workflows/default-workflow-function';
+
+if (RUN_INTEGRATION_TESTS) {
```

Comment:
I prefer using the test environment for our newer tests.

See [this](https://github.com/temporalio/sdk-typescript/commit/b6669fc9eb6c8613ee3f673a80f33b087584c082#diff-426424443d6004f9a1ad57cffeb1dde07cf735727acd8861e2147d6a0af828e0) for reference.

Reply:
We'll get better coverage this way and it'll allow us to easily run our tests against different server implementations in the future.

Reply:
(better OS coverage)

Reply:
You're right. Will fix.